tests/functional/init/test_init.py

```diff
@@ -639,8 +645,8 @@ def test_init_provided_project_name_and_skip_profile_setup(
         mock_get.return_value = [project.adapter.type()]
 
         # provide project name through the init command
-        run_dbt(["init", project_name, "-s"])
-        manager.assert_not_called()
+        run_dbt(["init", project_name, "--skip-profile-setup"])
+        assert len(manager.mock_calls) == 0
```

Comment:
Changed from `assert_not_called()` to asserting that `manager.mock_calls` is empty because the former didn't appear to ever fail when I tested it by hand, even when there _were_ calls to the mocks.